srv/salt/_modules/osd.py

```diff
@@ -107,12 +107,14 @@ def configured(**kwargs):
         devices = __pillar__['ceph']['storage']['osds']
         devices = _filter_devices(devices, **kwargs)
         devices = devices.keys()
-    if 'storage' in __pillar__ and 'osds' in __pillar__['storage'] and __pillar__['storage']['osds'] is not None:
+    if 'storage' in __pillar__ and 'osds' in __pillar__['storage'] and 
```

Comment:
multi-line conditions need brackets, otherwise this is a syntax error. See https://stackoverflow.com/questions/181530/styling-multi-line-conditions-in-if-statements